Projects/UOContent/Gumps/Props/PropsGump.cs

```diff
@@ -76,57 +45,15 @@ public class PropertiesGump : Gump
         private static readonly int TotalWidth =
             OffsetSize + NameWidth + OffsetSize + ValueWidth + OffsetSize + SetWidth + OffsetSize;
 
-        private static readonly int MaxHeight = OffsetSize + (EntryHeight + OffsetSize) * (MaxEntriesPerPage + 1);
-
         protected static readonly int BackWidth = BorderSize + TotalWidth + BorderSize;
-        protected static readonly int BackHeight = BorderSize + MaxHeight + BorderSize;
-
-        public static readonly string[] BoolNames = { "True", "False" };
-        public static readonly object[] BoolValues = { true, false };
 
-        public static readonly string[] PoisonNames = { "None", "Lesser", "Regular", "Greater", "Deadly", "Lethal" };
+        private static readonly string[] BoolNames = { "True", "False" };
+        private static readonly object[] BoolValues = { true, false };
 
-        public static readonly object[] PoisonValues =
+        private static readonly string[] PoisonNames = { "None", "Lesser", "Regular", "Greater", "Deadly", "Lethal" };
```

Comment:
I actually don't like that these are hard coded. Let's build this from PoisonImpl.
To do that, let's add a `Configure()` (which should execute after PoisonImpl).
Get the levels from `Poison.Levels` and build the PoisonNames/PoisonValues.

With that said, should BoolNames, BoolValues, PoisonNames, and PoisonValues live in `Server.Types`?

Reply:
Moved boolean and poison consts to `Types` in acfc5ed.
Let's address `Poison`  cleanup as a separate task.